### PR TITLE
Track `master` instead of `polkadot-v0.9.9`

### PR DESCRIPTION
**Pull request by HCastano**
It looks like the `polkadot-v0.9.9` branch is too old for us to use and succesfully
connect to Rococo. In this PR I'm updating Substrate, BEEFY, Polkadot, and Cumulus to
`master`.

I'm using the Statemine runtime and service as a reference. The code for them can be
found [here](https://github.com/paritytech/cumulus/tree/master/polkadot-parachains).

This, as usual, is a bit of a pain. Right now I have the runtime compiling fine, but the
node doesn't compile. ~~I think it might have to do with me using some newer versions of
Substrate, BEEFY, and Polkadot than what's specific in the Lockfile of [Cumulus `master`](https://github.com/paritytech/cumulus/commit/6c8f1f7b3cabef9c724772c76fb9adcb12c09506).~~

I've opened https://github.com/paritytech/cumulus/pull/602, which fixes some of the node issues
I was running into.

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -631,6 +691,12 @@ construct_runtime!(
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 11,
 
 		// Collator support. The order of these 4 are important and shall not change.
+		//
+		// TODO: Do we want to add these yet?
```
Comment (riusricardo) [inline runtime/src/lib.rs:695]: I don't see any reason for not adding it. It won't add any complexity in terms of development or testing.

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -355,16 +350,28 @@ impl pallet_timestamp::Config for Runtime {
 	type Moment = u64;
 	type OnTimestampSet = ();
 	type MinimumPeriod = MinimumPeriod;
-	type WeightInfo = ();
+	type WeightInfo = (); // TODO: Should this have a weight?
```
Comment (riusricardo) [inline runtime/src/lib.rs:353]: Yes, we need the custom weights. In order to get them we need to run the benchmarks.

**Review by riusricardo — COMMENTED** [review]
LGTM :+1:

**Review by cmichi — APPROVED** [review]
LGTM!

I tested it locally with `polkadot-launch`, works fine ‒ I can (still) deploy and interact with contracts on the parachain.

File: runtime/src/lib.rs (hunk 2 above)
Comment (cmichi) [reply runtime/src/lib.rs]: What's the process for this and who can we ping about it?

**Pull request merged**